spec/hydra/works/services/generic_file/generate/extracted_text_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
+
+describe Hydra::Works::GenerateExtractedText do
```

Comment:
This test should probably have a mock for FullTextExtractionService and not call it.